_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -6178,7 +6177,6 @@ mfxStatus ConfigureExecuteParams(
                 }
                 else if (MFX_EXTBUFF_VPP_SCENE_ANALYSIS == bufferId)
                 {
-                    executeParams.bSceneDetectionEnable = false;
                 }
```

Comment:
This empty 'else if' branch looks suspicious. I can't remove it without changing something else, because on some tests it falls to last else branch and MFX_ERR_INVALID_VIDEO_PARAM is produced.

Reply:
That's ok. Please, file another issue here on github requesting to clarify whether MFX_EXTBUFF_VPP_SCENE_ANALYSIS is actually supported (I suspect not).  We will let out VP team to handle that. Post a link here.

Reply:
Created #2053

Reply:
cool. I think current PR is done. +1.